src/commands/service/get-display-name.ts

```diff
@@ -0,0 +1,72 @@
+import commander from "commander";
+import { read as readBedrockYaml } from "../../lib/bedrockYaml";
+import { build as buildCmd, exit as exitCmd } from "../../lib/commandBuilder";
+import { logger } from "../../logger";
+import decorator from "./get-display-name.decorator.json";
+import { build as buildError, log as logError } from "../../lib/errorBuilder";
+import { errorStatusCode } from "../../lib/errorStatusCode";
+
+export interface CommandOptions {
+  path: string | undefined;
+}
+
+/**
+ * Executes the command, can all exit function with 0 or 1
+ * when command completed successfully or failed respectively.
+ *
+ * @param opts validated option values
+ * @param exitFn exit function
+ */
+export const execute = async (
+  opts: CommandOptions,
+  exitFn: (status: number) => Promise<void>
+): Promise<void> => {
+  // The assumption is that this command should be ran from the directory where bedrock.yaml exists
+  try {
+    if (!opts.path) {
+      throw buildError(
+        errorStatusCode.VALIDATION_ERR,
+        "service-get-display-name-path-missing-param-err"
+      );
+    }
+    const bedrockFile = readBedrockYaml(process.cwd());
+    if (!bedrockFile) {
+      throw buildError(
+        errorStatusCode.FILE_IO_ERR,
+        "service-get-display-name-bedrock-yaml-missing-err"
+      );
+    }
+    for (const serviceIndex in bedrockFile.services) {
+      if (opts.path === bedrockFile.services[serviceIndex].path) {
+        console.log(bedrockFile.services[serviceIndex].displayName);
+        await exitFn(0);
```

Comment:
i am not comfortable with having command to finish in a for loop. please have it as (or other ways)
```
const found = Object.keys(bedrockFile.services).find(serviceIndex =>
      opts.path === bedrockFile.services[serviceIndex].path;

if (found) {
   console.log(bedrockFile.services[serviceIndex].displayName);
    await exitFn(0);
}
```